coordinator/src/services/validation/auth-level.ts

```diff
@@ -0,0 +1,24 @@
+import * as Hapi from "@hapi/hapi"
+
+export function userHasValidAuth(headers: Hapi.Util.Dictionary<string>, authLevel: "user" | "application"): boolean {
+  if (requiresAuth()) {
+    if (authLevel === "user") {
+      return requestHasUserAuth(headers)
+    } else {
+      return requestHasAppAuth(headers)
+    }
+  }
+  return true
+}
+
+function requiresAuth() {
+  return process.env.SANDBOX !== "1"
+}
+
+function requestHasAppAuth(headers: Hapi.Util.Dictionary<string>): boolean {
+  return headers["nhsd-identity-authentication-method"]?.includes("application")
```

Comment:
In the Signing Service we've used this same header name to carry the AMR value, which is a similar but different thing. I think we should call this something else to avoid confusion

Reply:
ive gone with `NHSD-Identity-Authentication-Level`